### PR TITLE
LPD-2039 Avoid positioning errors in the diagram when using the Arabic language

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-2039)

### CODE REVIEW

**Comment by larissacribeiro** [comment]
just started reviewing 😄

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss [hunk 1]
```diff
@@ -24,7 +24,6 @@ hr {
 		font-weight: 600;
 		line-height: 1.3rem;
 		overflow: hidden;
-		padding-right: 4rem;
```
Comment (larissacribeiro) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss:27]: hey @igor-franca, just to confirm, this change is unrelated to the bug right?  I saw it was an necessary improvement to the nodes header, great work!

**Review by larissacribeiro — APPROVED** [review]
LGTM! 🎉

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss (hunk 1 above)
Comment (igor-franca) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss:27]: Correct, when we are using the Arabic language. The titles of the nodes were positioned incorrectly due to this style.

I realized that it wasn't necessary even for the other languages ​​and decided to remove it.

**Comment by igor-franca** [comment]
Hey @larissacribeiro, can you review the added test.

**Comment by igor-franca** [comment]
ci:test:sf

**Review by larissacribeiro — APPROVED** [review]

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
The error is not related, in another PR I was facing this same problem and achieved a successful result with insistence. 🍀

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
Waiting for: https://liferay.slack.com/archives/C1SJ64S9X/p1721932336510469

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:sf

**Review by rebsilva — APPROVED** [review]
LGTM! :cherry_blossom:

**Comment by igor-franca** [comment]
I saw that the formatter fix was merged by Brian, I am performing a rebase with Brian's Master to receive this fix.

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:forward:force
